js-basics/js-createelement_social-media-post/js/index.js

```diff
@@ -10,3 +10,51 @@ likeButton.addEventListener("click", handleLikeButtonClick);
 
 // Exercise:
 // Use document.createElement() and append another social media post to the body.
+
+const secondPost = document.createElement("section");
+secondPost.classList.add("post");
+
+const postContent = document.createElement("p");
+postContent.classList.add("post__content");
+postContent.textContent =
+  "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Lorem ipsum dolor, sit amet consectetur adipisicing elit.";
+
+const postFooter = document.createElement("footer");
+postFooter.classList.add("post__footer");
+
+const usernameSpan = document.createElement("span");
+usernameSpan.classList.add("post__username");
+usernameSpan.textContent = "@anotheruser";
+
+const likeButtonSecondPost = document.createElement("button");
+likeButtonSecondPost.classList.add("post__button");
+likeButtonSecondPost.setAttribute("type", "button");
+likeButtonSecondPost.setAttribute("data-js", "like-button");
+likeButtonSecondPost.textContent = "♥ Like";
+
+likeButtonSecondPost.addEventListener("click", handleLikeButtonClick);
```

Comment:
I would expect that you have to call the function with empty round bracets at the end [handleLikeButtonClick() ], but if it works it's all good.